test/stdlib/Json.test.js

```diff
@@ -0,0 +1,180 @@
+const { Interpreter } = require('../../lib/interpreter');
+const { FormatJson, ParseJson } = require("../../lib/stdlib/index");
+const { BrsArray } = require("../../lib/brsTypes/components/BrsArray");
+const { AssociativeArray } = require("../../lib/brsTypes/components/AssociativeArray");
+const {
+    BrsBoolean,
+    BrsInvalid,
+    BrsString,
+    Float,
+    Int32,
+    Int64,
+    Uninitialized
+} = require("../../lib/brsTypes");
+
+function withEnv(k, v, fn) {
+    let save = process.env[k];
+    try {
+        process.env[k] = v;
+        return fn();
+    } finally {
+        process.env[k] = save;
+    }
+}
+
+function expectConsoleError(expected, fn) {
+    jest.spyOn(console, 'error').mockImplementationOnce((s) => {
+        expect(s).toMatch(expected);
```

Comment:
Watch out! Because the assertion only happens during the stubbed call to `console.error`, any test that calls `expectConsoleError` will still pass!  Same with additional calls to `console.error`.  You'll likely want to have the assertion outside of the `mockImplementation` – I've got that working locally so I'll just push that in a second.